example/dj/urls.py

```diff
@@ -11,5 +13,7 @@
     url(r'^api/extra/$', ExtraResource.as_view())
 ]
 
-if django.get_version() < '1.9':
+if StrictVersion(django.get_version()) < StrictVersion('1.9'):
```

Comment:
spíš si myslím že si postavím knihocnu django-compatibility, která bude sloužit pro všechny knihovny, nicméně stejně tam budu rozlišovat co se pro kterou verzi vrací aby v tom byl pořádek. Jelikož tohle co píšeš ty je stračně tricky. Co kdy by se v djangu 1.11 rozhodli patterns přidat s tím že budou fungovat zas trochu jinak a budou sloužit k něčemu jinému? U patterns je to hodně hloupý příklad, ale v modelech se to už několikrát stalo..